movie-recommendations/.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
```

Comment:
I am not sure if we should ignore the .env files? how am I supposed to see them?